src/scraping/tools.py

```diff
@@ -253,3 +256,52 @@ def dataframe_to_db(self) -> None:
                     if_exists='replace',
                     index=False
                     )
+
+
+class GoogleSheets(DataBaseInteraction):
+    def __init__(self) -> None:
+        super().__init__()
+
+    @staticmethod
+    def base64_to_json(b64) -> dict[str, str]:
+        decodedbytes: bytes = base64.b64decode(b64[1:-1])
```

Comment:
I'll send the link to the named ranges and list slicing, e.g. https://www.w3schools.com/python/ref_func_slice.asp